src/angularComponent/ngcOmniboxController.js

```diff
@@ -16,4 +16,18 @@ export default class NgcOmniboxController {
       this.suggestions = suggestions;
     });
   }
+
+  hasSuggestions() {
+    const suggestions = this.suggestions;
+
+    if (!suggestions) {
+      return false;
+    } else if (Array.isArray(suggestions) && !suggestions.length) {
+      return false;
+    } else if (typeof suggestions === 'object' && !Object.keys(suggestions).length) {
+      return false;
+    }
+
+    return true;
```

Comment:
This is going to return `true` for strings, numbers, and the value `true` (meaning, `this.suggestions` is neither Array nor object. Please add a unit test for this method.